File: app/src/main/java/com/iamagamedev/mybrandnewgame/gameObjects/worldObjects/Forest.java
package com.iamagamedev.mybrandnewgame.gameObjects.worldObjects;

import com.iamagamedev.mybrandnewgame.gameObjects.GameObject;

/**
 * Created by Михан on 08.06.2017.
 */

public class Forest extends GameObject {

    public Forest(float worldStartX, float worldStartY, char type){
        final float HEIGHT = 1;
        final float WIDTH = 1;

        setHeight(HEIGHT);
        setWidth(WIDTH);

        setType(type);

        setBitmapName("wood");
        setBadBitmapName("wood");

        setWorldLocation(worldStartX, worldStartY, -1);
        setRectHitBox();
    }
    @Override
    public void update(long fps) {

    }
}

File: app/src/main/java/com/iamagamedev/mybrandnewgame/gameObjects/worldObjects/Forest.kt
package com.iamagamedev.mybrandnewgame.gameObjects.worldObjects

import com.iamagamedev.mybrandnewgame.constants.ObjectNames
import com.iamagamedev.mybrandnewgame.gameObjects.GameObject

/**
 * Created by Михан on 08.06.2017.
 */
class Forest(worldStartX: Float, worldStartY: Float, type: Char) : GameObject() {
    override fun update(fps: Long) {}

    init {
        val HEIGHT = 1f
        val WIDTH = 1f
        height = HEIGHT
        width = WIDTH
        this.type = type
        bitmapName = ObjectNames.WOOD
        badBitmapName = ObjectNames.WOOD
        setWorldLocation(worldStartX, worldStartY, -1)
        setRectHitBox()
    }
}